tests/test_language_en.py

```diff
@@ -0,0 +1,145 @@
+        ]
+    )
+    def test_parse_number_digits(self, expected, test_input):
+        assert parser.parse_number(test_input, LANG) == expected
+
+    @pytest.mark.parametrize(
+        "test_input,expected",
+        [
+            ("twenty-five cows, twelve chickens and one hundred twenty five kg of potatoes.",
+                "25 cows, 12 chickens and 125 kg of potatoes."),
+
+            ("I have eight cows",
+                "I have 8 cows"),
+
+            ("I have eight cows three bulls and seven hundred and twelve million dollars ",
+                "I have 8 cows 3 bulls and 712000000 dollars"),
+
+            ("They just won seventy-five thousand dollars",
+                "They just won 75000 dollars"),
+
+            ("I have eight cows. I don't have eighteen cows",
+                "I have 8 cows. I don't have 18 cows"),
+
+            ("thirty-four cows = thirty four cows",
+                "34 cows = 34 cows"),
+
+            ("thirty       four       cows = thirty four cows",
+                "34 cows = 34 cows"),
+
+            ("the     sun      is     hundred     and   twelve       km     away.",
+                "the     sun      is     112 km     away."),
+        ]
+    )
+    def test_basic_sentences(self, expected, test_input):
+        assert parser.parse(test_input, LANG) == expected
+
+    @pytest.mark.parametrize(
+        "test_input,expected",
+        [
+            ("one two three four twenty five.",
+                "1 2 3 4 25."),
+
+            ("one two three four twenty, five.",
+                "1 2 3 4 20, 5."),
+
+            ("three twenty seven.",
+                "3 27."),
+        ]
+    )
+    def test_ambiguity_in_separators(self, expected, test_input):
+        assert parser.parse(test_input, LANG) == expected
+
+    @pytest.mark.parametrize(
+        "test_input,expected",
+        [
+            ("OnE DaY at a Time.",
+                "1 DaY at a Time."),
+
+            ("SeVentY THREE days of SUMMER!!!.",
+                "73 days of SUMMER!!!."),
+
+            ("Twelve 11 pm",
+                "12 11 pm"),
+        ]
+    )
+    def test_case_of_string(self, expected, test_input):
+        assert parser.parse(test_input, LANG) == expected
+
+    @pytest.mark.parametrize(
+        "test_input,expected",
+        [
+            ("two thousand thousand",
+                "2000 1000"),
+
+            ("two thousand million",
+                "2000000000"),
+
+            ("two thousand two million",
+                "2002000000"),
+
+            ("two thousand, two million",
+                "2000, 2000000"),
+
+            ("two hundred thousand",
+                "200000"),
+
+            ("two thousand hundred",
+                "2100"),
+
+            ("two hundred hundred",
+                "200 100"),
+
+            ("million twenty three billion",
+                "1000023000000000"),
+
+            ("billion twenty three million",
+                "1000000023 1000000"),
```

Comment:
Shouldn't this be `1023000000` (1_023_000_000)?

Reply:
Yes thanks, will need to fix it , I realize what is the mistake. 

Another thing this made me realize is that this part of the logic for deciding multiplier validity is dependent on how multipliers are spaced. eg) In English all multipliers are spaced by 10^3, 
``` thousand (10^3) ,  million (10^6),  billion(10^9).``` (However sometimes billion can mean (10^12) too.
 
However, I will change this and look to implement a language independent code. 
I will test this using ```en-IN``` locale as in India we have ```lakhs (1,00,000)``` and ```crore(1,00,00,000).``` Thus spaced by 10^2. 

For eg-
```en locale -> parse("one hundred thousand and fifty thousand") ==  "100050 1000"```
```en-IN locale -> parse("one lakh and fifty thousand") == 150000```